spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -61,6 +61,8 @@ public ListenableFuture<String> publish(String topic, byte[] payload, Map<String
 public ListenableFuture<String> publish(String topic, ByteString payload, Map<String, String> headers)
 
 public ListenableFuture<String> publish(String topic, PubsubMessage pubsubMessage)
+
+public <T> ListenableFuture<String> publish(String topic, T payload, Map<String, String> headers) throws JsonProcessingException;
```

Comment:
Can probably leave the `throws ...` out of the docs.

Reply:
Not only from docs: I wouldn't stick just only with JSON at all.
Therefore it would be pretty confusing to me to do some other configuration, but see a JSON-specific exception in the contract.

Reply:
done.